test/SPIRV/llvm.memmove.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s
+
+; CHECK-NOT: llvm.memmove
+
+; CHECK: Variable [[voidPtr:[0-9]+]] [[mem:[0-9]+]] 7
```

Comment:
There is no need in voidPtr variable - it's not used in the test. Suggest replacing with {{[0-9]+}}.